cpp/src/gandiva/function_registry_string.cc

```diff
@@ -209,6 +209,71 @@ std::vector<NativeFunction> GetStringFunctionRegistry() {
                      "convert_replace_invalid_fromUTF8_binary",
                      NativeFunction::kNeedsContext),
 
+      NativeFunction("convert_toDOUBLE", {"convert_todouble"}, DataTypeVector{float64()},
```

Comment:
Can you add a comment explaining convert_toX functions converts X type to binary

Reply:
I add a comment in the place that functions are created